dokku

```diff
@@ -207,14 +210,22 @@ case "$1" in
     ;;
 
   plugins)
-    ls -1 -d $PLUGIN_PATH/*/
+    for plugin in "$PLUGIN_PATH"/*/; do
+      basename "$plugin"
+    done
     ;;
 
   plugins-install)
```

Comment:
I think `plugins-install` (and `plugins-install-dependencies`, `plugins-update`) should be renamed `plugins:install` to follow the convention used everywhere else.